lib/remove.js

```diff
@@ -1,46 +1,45 @@
 'use strict'
 
-const fs = require('fs')
-const rm = require('rimraf')
+const fs = require('fs').promises
```

Comment:
`require('fs/promises')`

Reply:
or maybe 'graceful-fs'?

Reply:
This has been addresses by changing to `fs/promises` in all relevant files.